[LTP] Enable select, pselect, poll, epoll tests

dimakuv opened the pull request:
Graphene now has a reworked select/poll/epoll mechanisms, and corresponding LTP tests pass now (except a couple with issues unrelated to polling).

For more info, see descriptions in https://github.com/oscarlab/graphene/issues/1117, in particular:
- https://github.com/oscarlab/graphene/issues/1117#issuecomment-549095106
- https://github.com/oscarlab/graphene/issues/1117#issuecomment-549188771
- https://github.com/oscarlab/graphene/issues/1117#issuecomment-549563732

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene-tests/63)
<!-- Reviewable:end -->

mkow reviewed (dismissed):
> Reviewed 5 of 5 files at r1.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene-tests/63)* status: all files reviewed, all discussions resolved, not enough approvals from maintainers (1 more required)



<!-- Sent from Reviewable.io -->

yamahata reviewed (dismissed):
> Reviewed 5 of 5 files at r1.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene-tests/63)* status: :shipit: complete! all files reviewed, all discussions resolved



<!-- Sent from Reviewable.io -->

yamahata reviewed (approved):
> Reviewed 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene-tests/63)* status: all files reviewed, all discussions resolved, not enough approvals from maintainers (1 more required), not enough approvals from different teams (1 more required, approved so far: Intel)



<!-- Sent from Reviewable.io -->

mkow reviewed (approved):
> Reviewed 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene-tests/63)* status: :shipit: complete! all files reviewed, all discussions resolved



<!-- Sent from Reviewable.io -->

Commit:
[LTP] Enable select, pselect, poll, epoll tests

Graphene now has a reworked select/poll/epoll mechanisms, and corresponding
LTP tests pass now (except a couple with issues unrelated to polling).
